.sync/azure_pipelines/matrix_dependent/Ubuntu-GCC5.yml

```diff
@@ -23,6 +23,9 @@ resources:
       endpoint: microsoft
       name: microsoft/mu_devops
       ref: refs/tags/{{ sync_version.mu_devops }}
+  containers:
+    - container: linux-gcc
+      image: ghcr.io/tianocore/containers/fedora-35-build:2113a0e

```

Comment:
nit: This is calls for a fedora 35 but the name of the file is Ubuntu....

Reply:
Yeah, I think using the container we should probably distance ourselves some a specific SKU in the naming. I want to go through and rename all such files to Linux-GCC.yml. I will likely do this after some of the dust settles a bit as it will require editing all the pipeline configurations as well.

Reply:
I don't think it really matters since the VM images are Ubuntu and likely will be for the foreseeable future. As long as the VM image and the container image names line up, I don't see a need for extra repo thrash.

Reply:
in fact, it has to be ubuntu for the pipeline container support from the documentation. Either way, we can continue this discussion offline but i'll resolve this thread for now.